frontend/components/Filter/FilterIcon.tsx

```diff
@@ -40,14 +38,15 @@ export default function FilterIcon(props: FilterIconProps) {
             alt="volleyballImage"
             className={`${props.style} ${
               props.selectedSport === ""
-                ? "grayscale-0"
+                ? "opacity-100"
                 : props.selectedSport === props.sportIdentifierString
-                ? "grayscale-0"
-                : "grayscale"
+                ? "opacity-100"
+                : "opacity-50"
             } flex justify-center`}
           />
         </div>
-        <p className="text-sm font-light grow text-center">{props.name}</p>
+        <p className="text-xs font-light grow text-center mb-1.5">{props.name}</p>
+        {props.selectedSport === props.sportIdentifierString && <div className="bg-black h-[2px]"></div>}
```

Comment:
Like airbnb's filter, I wonder if it would be possible to make it so that when hovering over sport filter, it gives indication of hovering and that it is clickable (darkens and underlines the icon)
<img width="50" alt="image" src="https://github.com/user-attachments/assets/a6cb1aaf-10dc-4929-96fe-e5c555e68899" />